src/components/Composer/index.android.tsx

```diff
@@ -121,24 +110,16 @@ function Composer({shouldClear, onClear, isDisabled, maxLines, forwardedRef, isC
             maxNumberOfLines={maxNumberOfLines}
             textAlignVertical="center"
             style={[composerStyles]}
+            readOnly={isDisabled}
+            autoFocus={autoFocus}
+            selection={selection}
+            isFullComposerAvailable={isFullComposerAvailable}
             /* eslint-disable-next-line react/jsx-props-no-spreading */
             {...props}
-            readOnly={isDisabled}
```

Comment:
We should leave `readOnly` after `{...props}`, maybe there was a reason for that.

Reply:
Right, updated!